igvm/vm.py

```diff
@@ -455,11 +455,10 @@ def run_puppet(self, clear_cert=False, debug=False):
         if clear_cert:
             with settings(
                 host_string=self.dataset_obj['puppet_ca'],
-                user='root',
                 warn_only=True,
             ):
                 run(
-                    '/usr/bin/puppet cert clean {}'.format(self.fqdn),
+                    'sudo /usr/bin/puppet cert clean {}'.format(self.fqdn),
```

Comment:
Please never directly type `sudo` in commands, we have a parameter of `run()` for that.